Makefile

```diff
@@ -0,0 +1,8 @@
+build: deps
+	npm run build
+
+deps:
+	npm install
+
+watch:
+	parcel build/index.js
```

Comment:
just to keep in mind that this will only watch for whatever you've built with pulp -w build --to build/index.js, not psc-ide crap in general. if you want editor changes with psc-ide to parcel build in 100-200ms whatever, then you want to use parcel output/FrontEnd